### DISCUSSION TITLE
Any reason this couldnt be installed on a Pi4?

### DISCUSSION THREAD

**Discussion by kachapman** [AdnanHodzic/auto-cpufreq#179] [Ideas]
I have a NextCloudPi server that I want running as efficiently as possible and love the results on my laptops. 
Will this cause any issues on a Pi? I know that is now compatible with Arm. 

Thanks!

**Comment by AdnanHodzic** [comment]
It should work just fine, I recommend you install and it first run it in ["monitor"](https://github.com/AdnanHodzic/auto-cpufreq/#monitor) or ["live" ](https://github.com/AdnanHodzic/auto-cpufreq/#live)mode, if you're happy with results perform install.
